extensions/cache/src/main/java/org/radargun/stages/cache/test/CacheTestStage.java

```diff
@@ -24,49 +24,27 @@
 
    @Property(doc = "Generator of keys used in the test (transforms key ID into key object). By default the generator is retrieved from slave state.",
```

Comment:
The doc for the next three properties needs to be updated to reflect the new default before the state is checked